modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -64,53 +70,67 @@ http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/m
-. Click the frequency of a polling element to open the schedule editor:
+. Click *Applications* in the left pane, click the app, and then click *Schedules*:
 +
-image::scheduleschangetime.png[SchedulesChangeTime]
+image::ch-schedule-change.png[View Schedules]
 +
-. In the schedule editor, adjust the polling frequency of the element, or disable the polling frequency by unchecking *Enabled*:
+. Select the elements to manage.
+
+* Click *Run now* to override the schedule and run the job immediately.
 +
-image::chsetpollfreq.png[CHSetPollFreq]
+This message appears: `The schedules have been queued to run.`
+* Click *Disable* to suspend a polling element.
 +
-. Click *Switch to advanced mode* to use Cron expressions, and click *Quick reference* for information on required fields and allowed values.  Click *more* to view the Quartz http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/tutorial-lesson-06.html[CronTrigger Tutorial]. If you change the Cron setting, click *Update*. Click *Cancel* to exit. Click *Switch to basic mode* to exit advanced mode.
+CloudHub does not run the polling job until you reenable it.
+. In the *Schedule* column, click the frequency of a polling element to open the schedule editor:
 +
-image::chadvmodewithqkref.png[CHAdvModeWithQkRef]
+image::ch-set-poll-freq.png[Set Polling Frequency]
 +
-[NOTE]
-All schedules are based in the UTC timezone, regardless of what region your CloudHub workers are on, and the timezone configuration is ignored.
+. In the schedule editor, change the polling frequency of the element or deselect *Enabled* to disable the polling schedule.
++ 
+You can also <<Use Cron Expressions,use cron expressions>> to create more specific schedules, such as "every Thursday at 3:00 pm". 
+. Click *Update* to save changed settings or click *Cancel* to exit without saving.
+. Click *Logs* in the left navigation menu to verify that the job you selected ran as expected.
 
-. To see the *Schedule* tab in action, select one or more of the scheduled jobs and click *Run Now*. `The schedules have been queued to run` message appears.
+=== Use Cron Expressions
+
+With cron expressions, you can create more complex polling frequencies, such as "every 5 minutes between 9:00 am and 10:00 on Fridays".
+
+To use cron expressions to specify the polling frequency:
+
+. In the schedule editor, click *Switch to advanced mode*.
+. In the advanced mode page, click *Quick reference* to expand the table that lists required fields and allowed values:
 +
-image::schedulesrunnow.png[SchedulesRunNow]
+image::ch-schedule-adv-mode-exp.png[Schedule Advanced Mode Expanded]
 +
-. To check the result, click *Logs* in the left navigation menu, and view the entries to verify that the job you selected ran as expected: +
+The cron expression in this example sets the polling schedule to every 10 seconds.
 +
-image::scheduleslogs.png[SchedulesLogs]
+On this page:
 
```

Comment:
The following items are choices and not chrono?

Reply:
Yes.